NEWS

```diff
@@ -13,6 +13,8 @@ Current git version
     clients ('floatplacement=center') or leaves the floating position as is
     ('floatplacement=none')
   * New rule condition 'pgid'
+  * Bug fixes:
+    - Fix wrong behaviour in 'cycle_layout'
```

Comment:
Could me a bit more verbose (without reading the corresponding commit message I wouldn't know what happened here).

Reply:
I don't know how it helps, since the bug is fixed now. But I clarified it a bit.

Reply:
Thank you!